includes/html/graphs/bill/historicmonthly.inc.php

```diff
@@ -4,9 +4,10 @@
 use Amenadiel\JpGraph\Plot\BarPlot;
 use Amenadiel\JpGraph\Plot\GroupBarPlot;
 use Amenadiel\JpGraph\Plot\LinePlot;
+use LibreNMS\Billing;
 use LibreNMS\Util\Number;
 
-$graph_data = getHistoricTransferGraphData($vars['id']);
+Billing::getHistoricTransferGraphData($vars['id']);
```

Comment:
Missing variable assignment?
Probably causing https://github.com/librenms/librenms/issues/14994

Reply:
Hmm, I checked all the graphs, thanks.